tests/language/explicit-resource-management/using-basic.js

```
@@ -151,4 +164,102 @@ describe("using declaration", () => {
     }
     expect(order).toEqual(["b", "a"]);
   });
+
+  test("using in switch case disposes on break", () => {
+    const order = [];
+
+    switch (1) {
+      case 1:
+        using resource = {
+          [Symbol.dispose]() { order.push("dispose"); }
+        };
```

Biome (2.4.13)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)